models/concerns/as_json.rb

```diff
@@ -20,9 +20,9 @@ module VrpAsJson
 
   def empty_json(options = nil)
     vrp = {
-      'name' => self.name,
-      'configuration' => self.configuration.as_json(options),
-      'points' => [], 'services' => [], 'vehicles' => []
+      name: self.name,
+      configuration: self.configuration.as_json(options),
+      points: [], services: [], vehicles: []
     }
 
     vrp.to_json
```

Comment:
the hash -> call to_json -> call Json.parse -> hash logic as been removed now. 

having empty_vrp within vrp class sounds a bit odd. Having it related to as_json in a concern does not shock me.

Reply:
Hmm, you are right about the oddness but having this custom logic in `as_json` is quite odd as well. Plus, in either case, this is not an empty json nor an empty vrp. Its name is generic (empty_json) but if we include the concern anywhere else (i.e., vehicles, services) it would not work. 

Instead of this, I suggest we save us the trouble and just do the following which prevents any confusion and actually less code (1 line). Since it is used only in one location, as long as we don't need this logic at +10x location we are better off.

```ruby
Models::VRP.create({ name:  original.name, configuration: original.configuration.as_json }) 
```
I assume the points, services, vehicles, etc will be automatically initialized to empty arrays, right?